_workinggroups/dataanalysis.md

```diff
@@ -3,22 +3,8 @@ title: "Data Analysis"
 layout: plain
 ---
 
-The Data Analysis Working Group considers developments and approaches
-that will make the analysis of HEP data towards final physics results
-faster and more scalable in the future. One new approach to be
-considered is the development of expressive functional interfaces for
-analysis. The meshing of HEP specific tools with those found in the
-larger data science community is another area of interest, where
-database approaches or formats optimised for high-speed processing may
-be profitable. The development of test beds for high performance
-analysis clusters, that may speed-up turn-around and efficiency is a
-major area of R&D. The group should help coordinate activities and
-communication here between experiments and IT experts. The integration
-of such systems into the end-to-end data flow, with caching and
-distribution policies, are important. Optimising the actual content of
-analysis formats is a boundary condition to discuss with physics groups,
-but can bring major gains.
+The Data Analysis Working Group aims to eliminate time spent by physicists working on monotonous and laborious tasks while performing physics analysis, and make the publication of physics results efficient both in terms of human and computing resources.  To this end, the group will prioritise capturing the requirements of physics analysis across the HEP domain, aiming to gain a detailed understanding of the needs of physicists by direct consultation. Participation of those people actively involved in writing data analysis software, especially those with experience of publishing several analyses, will be critical to the group’s success. As a close second priority, the group will work with pioneers of technology, both inside and outside of the HEP community. The goal is to build bridges between physics analysis experts and technical innovators, fostering collaboration between these two communities to produce optimal solutions to a well-defined set of problems.
```

Comment:
"time spent" -> "the time spent"
"and make" -> "and to make"

Reply:
Thanks guys! 

I have one additional criticism, which is that reading this it seems like you're starting from zero. I would add some mention of the work done in the CWP and maybe also the Analysis Ecosystem Review. Some phrase like "building on previous work..." would be enough.

Reply:
This is also a mega-line, which makes commenting more awkward. Can you split it up? I would suggest the traditional 80 character warp.